assets/js/vue/components/research-group/PeopleTab.vue

```diff
@@ -32,4 +46,8 @@
             height: 200px;
         }
     }
+
+    .text-width {
```

Comment:
This may not be the best class name, it's kinda ambiguous, not very descriptive of what it goes with.
Maybe it should be `organization-max-width` ?

Reply:
yes, you are right. Changed it